static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -71,9 +76,28 @@ export function getCoverageColors(
       case Coverage.NOT_APPLICABLE:
       // fallthrough
       default:
+        if (lineNo === primaryLineNumber) {
+          primaryLineCovered = false;
+        } else {
+          surroundingLinesCovered = false;
+        }
         return 'transparent';
     }
   });
+  if (organization) {
+    trackAdvancedAnalyticsEvent('issue_details.codecov_primary_line_coverage_shown', {
+      organization,
+      success: primaryLineCovered,
+    });
+    trackAdvancedAnalyticsEvent(
+      'issue_details.codecov_surrounding_lines_coverage_shown',
+      {
+        organization,
+        success: surroundingLinesCovered,
+      }
+    );
```

Comment:
this is going to submit tons of analytic events, this probably isn't what we want

Reply:
Is there a way we could include the coverage info in the existing "Issue Details: Viewed" event? My hunch is no, but that would be ideal

Reply:
That's a good idea, there is `useRouteAnalyticsParams` which might be what you're looking for

Reply:
thank you! I'll keep looking into this, to see how easily it can be done. In the meantime, I put up a solution with `useMemo` in case we want this feature quickly